exercises/combining_variables/README.rst

```diff
@@ -0,0 +1,52 @@
+Multivariable Inputs and vertical integration
+---------------------------------------------
+
+Here we show how use two input variables and perform a
+vertical integration. The motivating example is from the
+``CMIP6_Omon.json`` table:
+
+  .. code:: json
+
+      "intpp": {
+          "standard_name": "net_primary_mole_productivity_of_biomass_expressed_as_carbon_by_phytoplankton",
+          "units": "mol m-2 s-1",
+          "long_name": "Primary Organic Carbon Production by All Types of Phytoplankton",
+          "comment": "Vertically integrated total primary (organic carbon) production by phytoplankton.  This should equal the sum of intpdiat+intpphymisc, but those individual components may be unavailable in some models.",
+   }
+
+In the ``AWI-ESM-1-REcoM`` case two phytoplanktion groups
+contribute to primary production:
+
+1. nanophytoplankton (variable ``diags3d01``)
+2. diatoms (variable ``diags3d02``)
+
+Hence, the workflow to get implemented would be:
+
+.. math::
+
+    \text{NPP} = \int_{\text{bottom}}^{\text{top}} (\text{NPP}_{\text{nanopythoplankton}} + \text{NPP}_{\text{diatoms}})
```

Comment:
The math is not working here right?

Reply:
That might be a limitation of how reST works. I had just copy-pasted the example out of the main repo, maybe it is better to have a separate one just for the workshop.